sources/packages/api/src-sql/Institution/Rollback-drop-col-legal-authority-contact.sql

```diff
@@ -0,0 +1,14 @@
+-- Add legal_authority_contact to institutions table
+ALTER TABLE
```

Comment:
regarding the file name its `add` instead of `drop`

Reply:
Drop is my main action, Rollback drop is my rollback action.

Reply:
mm, I didn't get it, your main action in this rollback is adding the column back right? Not sure, if I capture your point correctly
![image](https://user-images.githubusercontent.com/77353155/158675810-6b6b7cf6-22c9-497a-9ff4-074f5dbf5e12.png)

Reply:
file name pattern I follow here is, main-action.sql(drop-col.sql)
rollback-main-action.sql(rollback-drop-col.sql) rollback of drop col is add col.